packages/api/tests/session.test.ts

```diff
@@ -236,4 +236,8 @@ describe('session', () => {
     expect(sessions[1]).toEqual(undefined)
     expect(client.sessionManager.active()).toEqual(false)
   })
+
+  it('reuses the client object across calls', () => {
+    expect(client).toEqual(sessionClient.service(server.url))
```

Comment:
I think we want `toBe` here, which checks for shallow equality (`===`).
```suggestion
    expect(client).toBe(sessionClient.service(server.url))
```